test/functions.php

```diff
@@ -17,9 +17,10 @@ public function testextractDataAndDisplaySuccess()
             ]
         ];
         $expected_success = '<section class="container-box"><div class="sub-box"><img src="img/string" alt="string">'
-                            .'<h2 class="title">string</h2><h4 class="artist">Artist: string</h4>'
-                            . '<h5 class="type">Type: string</h5>'
-                            .'<p class="descripton">Description: </p><p>string</p></div></section>';
+                            . '<h2 class="title">string</h2>'
+                            . '<h4 class="artist">Artist: string</h4><h5 class="type">Type: string</h5>'
```

Comment:
put `<h5>` on a new line to be consistent with the other elements, which are on their own lines.